project/Build.scala

```diff
@@ -488,6 +488,10 @@ object ScaldingBuild extends Build {
     previousArtifact := None,
     crossPaths := false,
     autoScalaLibrary := false,
+    // Disable cross publishing for this artifact
+    publishArtifact <<= (scalaVersion) { scalaVersion =>
+        if(scalaVersion.startsWith("2.11")) false else true
```

Comment:
when we move to 2.11 + 2.12, this will probably need to be updated. but as will many other things.